src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,65 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import React from 'react';
+import data from '../../public/api/people.json';
```

Comment:
Importing data directly from a JSON file within the component can lead to performance issues, especially if the data is large. Consider fetching the data asynchronously or passing it as a prop to the component.